LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini

```diff
@@ -425,10 +596,12 @@ BULLPUP_COIL_IENVIRONMENTDAMAGE=5
 BULLPUP_COIL_UPGRADESLOTS=2
 
 AssaultRifle_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilRifle_Base"
+BattleRifle_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilRifle_Base"
 SMG_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilSMG_Base"
 Cannon_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilCannon_Base"
 Shotgun_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilShotgun_Base"
 SniperRifle_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilSniperRifle_Base"
+MarksmanRifle_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilSniperRifle_Base"
```

Comment:
Did realitymachina's pack not have these assets? If not I'll need to put in an art request. Let me know